setup.py

```diff
@@ -14,10 +18,17 @@
 # python3.4 setup.py bdist_wheel upload
 # python3.4 -m pip install
 
+here = os.path.dirname(__file__)
+
+def read(fname, encoding='utf-8'):
+    with io.open(os.path.join(here, fname), encoding=encoding) as f:
+        return f.read()
+
 setup(
     name='vstruct',
     version='2.0.2',
     description='Vivisect Structure Definition/Parsing Library',
+    long_description=read('README.rst'),
```

Comment:
Digging this idea, but not super familiar with reST.  What do we lose by moving to it?  Are most of the crazy markdown formatting/conversion tools available for reST?